backend/application/syson-sysml-import/src/main/java/org/eclipse/syson/sysml/parser/ReferenceHelper.java

```diff
@@ -0,0 +1,94 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.syson.sysml.parser;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.eclipse.syson.sysml.AstConstant;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.Streams;
+
+/**
+ * ReferenceHelper.
+ *
+ * @author gescande.
+ */
+public class ReferenceHelper {
+
+    //Node  , AstConstant.TARGET_CHAIN_CONST, , AstConstant.RESULT_CONST, AstConstant.SOURCE_CONST,AstConstant.THEN_CONST, AstConstant.ACCEPTER_CONST, AstConstant.GUARD_CONST, AstConstant.EFFECT_CONST, AstConstant.RECEIVER_CONST, AstConstant.PAYLOAD_CONST,
```

Comment:
Please remove all comments like this one

Reply:
Ok